### ISSUE TITLE
Column group headers are not accessible via keyboard navigation

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#6595]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=24600&p=122153#p122153)

"Hello Bryntum team
Currently the column group headers cannot be navigated onto with a keyboard and I have also been unable to get a screen reader to read out the column group text. This can be seen on your "Collapsible group headers demo". Please can you make the group headers accessible."

Check the video for example, you can't access the "Air Quality Metrics" or "Meta" headers via keyboard navigation.

https://user-images.githubusercontent.com/16693227/232072091-fa6cbb2b-51e2-4433-be0e-b761e95fd255.mp4

**Comment by amurashincertinia** [comment]
Hello there!

I've just tested this in v7 demos and can confirm that this issue is still relevant.

Despite the parent column text now being included in the screen reader announcement, keyboard focus still does not go into the parent column element, making it inaccessible to keyboard users. For example, it is not possible to collapse or expand a column group without using a mouse/trackpad/etc.

Thanks!
